pilot/pkg/kube/inject/inject.go

```diff
@@ -236,6 +242,30 @@ func (p *Params) Validate() error {
 	return ValidateExcludeInboundPorts(p.ExcludeInboundPorts)
 }
 
+// intoHelmValues returns a map of the traversed path in helm values YAML to the param value.
+func (p *Params) intoHelmValues() map[string]string {
+	vals := map[string]string{
+		"global.proxy_init.image":                    p.InitImage,
+		"global.proxy.image":                         p.ProxyImage,
+		"global.proxy.enableCoreDump":                strconv.FormatBool(p.EnableCoreDump),
+		"global.proxy.privileged":                    strconv.FormatBool(p.Privileged),
+		"global.imagePullPolicy":                     p.ImagePullPolicy,
+		"global.proxy.statusPort":                    strconv.Itoa(p.StatusPort),
+		"global.proxy.tracer":                        p.Tracer,
+		"global.proxy.readinessInitialDelaySeconds":  strconv.Itoa(int(p.ReadinessInitialDelaySeconds)),
+		"global.proxy.readinessPeriodSeconds":        strconv.Itoa(int(p.ReadinessPeriodSeconds)),
+		"global.proxy.readinessFailureThreshold":     strconv.Itoa(int(p.ReadinessFailureThreshold)),
+		"global.sds.enabled":                         strconv.FormatBool(p.SDSEnabled),
+		"global.sds.useTrustworthyJwt":               strconv.FormatBool(p.EnableSdsTokenMount),
+		"global.proxy.includeIPRanges":               p.IncludeIPRanges,
+		"global.proxy.excludeIPRanges":               p.ExcludeIPRanges,
+		"global.proxy.includeInboundPorts":           p.IncludeInboundPorts,
```

Comment:
Why you removed the p.IncludeInboundPorts value set in  https://github.com/istio/istio/pull/10830/files#diff-660cae0de46bea17fa1f553d7c030f31L296, but still use it here?

Reply:
Because i don't want to break unit tests.

Reply:
iiuc, this is useless, can u clean up and fix the tests?

Reply:
After I removed hardcoded template, the [tests](https://github.com/istio/istio/blob/master/pilot/pkg/kube/inject/inject_test.go#L65) read input from `install/kubernetes/helm` instead of original template. This map aims to create mappings between variables in `install/kubernetes/helm` and struct [Params](https://github.com/istio/istio/blob/master/pilot/pkg/kube/inject/inject_test.go#L510).